extensions/functions_string.yaml

```diff
@@ -197,6 +197,19 @@ scalar_functions:
           dotall:
             values: [ DOTALL_DISABLED, DOTALL_ENABLED ]
         return: "string"
+      - args:
```

Comment:
this addition doesn't match the description. do in separate definition with separate description.